os/test_fw/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config KERNEL_TEST_FW
+	bool "Kernel Test Framework Support"
```

Comment:
As I said before, this config should be enabled and disabled by app's testcase config automatically. Just sourcing (including) Kconfig from apps is not my opinion.

Reply:
Ok. Added dependency to auto select KERNEL_TEST_FW based on EXAMPLES_TEST_KERNEL and EXAMPLES_TEST_DRIVER.